docs/2.guide/4.recipes/4.sessions-and-authentication.md

```diff
@@ -0,0 +1,403 @@
+---
+title: 'Sessions and Authentication'
+description: "User registration and authentication is an extremely common requirement in web apps. This recipe will show you how to implement basic user registration and authentication in you Nuxt app."
+---
+
+
+## Introduction
+
+In this recipe we'll be setting up user registration, login, sessions, and authentication in a full-stack Nuxt app.
+ We'll be using [Nuxt Auth Utils](https://github.com/Atinux/nuxt-auth-utils) by [Altinux (Sébastien Chopin)](https://github.com/Atinux) which provides convenient utilities for managing front-end and back-end session data. We'll install and use this to get the core session management functionality we're going to need to manage user logins. For the database ORM we'll be using [Drizzle](https://orm.drizzle.team/) with [db0](https://db0.unjs.io/), but you can use any ORM or database connection strategy you prefer.
+
+You'll need a `users` table in your database with the following columns:
+- `id` (int, primary key, auto increment)
+- `email` (varchar)
+- `password` (varchar)
+
+Additionally, we'll use [nuxt-aut-utils](https://github.com/Atinux/nuxt-auth-utils) by [Atinux](https://github.com/Atinux) to handle the authentication and session management.
+
+## Steps
+
+### 1. Install nuxt-auth-utils
+
+
+Install the [auth-utils](https://github.com/Atinux/nuxt-auth-utils) module using the `nuxi` CLI.
+
+
+```bash
+npx nuxi@latest module add auth-utils
+```
+
+### 1a. (Optional) Add a session encryption key
+
+Session cookies are encrypted. The encryption key is set from the `.env` file. This key will be added to your `.env` automatically when running in development mode the first time. However, you'll need to add this to your production environment before deploying.
```

Comment:
This sentence seems a bit too.. "hard", there are a lot of stops here, might be worth rewording into something like:
```md
Session cookies are encrypted using a key set from the `.env` file. This key will be added to your `.env` automatically when running in development mode the first time.
```

Reply:
Updated. This is very good feedback on readability and flow.

Reply:
Glad I can help!

Reply:
@Smef 
Actually, just about about the `set` that is also redundant, as the next sentence talks about the key setter, so it can stay 
```md
using a key from the `.env` file.
```

Reply:
Updates